mautic/mautic
# Close memory leak and use a limit to prevent script from maxing it out

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

There was a memory leak due to Doctrine holding onto the LeadEventLog entity. Also added a limit option to the command to prevent the script from maxing out memory. 

[//]: # ( As applicable: )
#### Steps to test this PR:
1. Import a CSV
2. Run the `app/console mautic:import --limit=LIMIT` and replace LIMIT with a low number
3. The import should import LIMIT number of records and then the import get placed on DELAY
4. Run the command again with --limit and it should import the rest
